enterprise/app/api_keys/api_keys.tsx

```diff
@@ -261,7 +261,7 @@ export default class ApiKeysComponent extends React.Component<ApiKeysComponentPr
   }
 
   private onSelectSCIM(onChange: (name: string, value: any) => any) {
-    onChange("capability", [api_key.ApiKey.Capability.SCIM_CAPABILITY]);
+    onChange("capability", [api_key.ApiKey.Capability.ORG_ADMIN_CAPABILITY]);
```

Comment:
The UI text on line 382 also needs to be updated.

Reply:
Done